console_conf/controllers/identity.py

```diff
@@ -116,10 +117,21 @@ def write_login_details(fp, username, ips):
         )
     else:
         first_ip = ips[0]
+        key_info = None
+        if os.getenv("SNAP_CONFINEMENT", "classic") == "strict":
+            # if we run in confinement, we have no direct accesss to host
+            # keys info use prepared finger prints if exist
+            host_fingerprints_path = "/run/console-conf/host-fingerprints.txt"
```

Comment:
Let's use `SNAP_NAME` instead of hardcoding console-conf, this will help future confined Subiquity efforts.